<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>900</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string notr="true">Kenshi Mod Manager</string>
  </property>
  <property name="locale">
   <locale language="English" country="Netherlands"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>400</width>
      <height>600</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>398</width>
       <height>598</height>
      </rect>
     </property>
     <widget class="QListWidget" name="listWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>400</width>
        <height>600</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::CurrentChanged|QAbstractItemView::EditKeyPressed|QAbstractItemView::SelectedClicked</set>
      </property>
      <property name="dragEnabled">
       <bool>true</bool>
      </property>
      <property name="dragDropMode">
       <enum>QAbstractItemView::InternalMove</enum>
      </property>
      <property name="defaultDropAction">
       <enum>Qt::TargetMoveAction</enum>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::ExtendedSelection</enum>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="playButton">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>531</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Play</string>
    </property>
   </widget>
   <widget class="QPushButton" name="configButton">
    <property name="geometry">
     <rect>
      <x>575</x>
      <y>531</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Save load order</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="rootPath">
    <property name="geometry">
     <rect>
      <x>455</x>
      <y>70</y>
      <width>400</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="workshopPath">
    <property name="geometry">
     <rect>
      <x>455</x>
      <y>200</y>
      <width>400</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="rootLabel">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>20</y>
      <width>391</width>
      <height>51</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::PreventContextMenu</enum>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Root Path&lt;/span&gt;&lt;br/&gt;&lt;br/&gt;Put the root folder of kenshi here (Where the exe of kenshi is located) &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="workshopLabel">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>99</y>
      <width>391</width>
      <height>100</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::PreventContextMenu</enum>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Workshop Path&lt;br/&gt;&lt;/span&gt;&lt;br/&gt;Put the folder where your kenshi workshop mods are located, usually it's in:&lt;/p&gt;&lt;p&gt;If you don't use steam for kenshi you can leave this empty.&lt;/p&gt;&lt;p&gt;&amp;lt;WhereSteamIs&amp;gt;/steamapps/workshop/content/233860&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pathButton">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>531</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Save paths</string>
    </property>
   </widget>
   <widget class="QLabel" name="steamLabel">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>219</y>
      <width>391</width>
      <height>97</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::PreventContextMenu</enum>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Steam Path&lt;br/&gt;&lt;/span&gt;&lt;br/&gt;Path to where your steam folder is located. &lt;/p&gt;&lt;p&gt;This is for the play function of the mod manager.&lt;/p&gt;&lt;p&gt;(If you rather open the game via steam itself you can leave this empty)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="steamPath">
    <property name="geometry">
     <rect>
      <x>455</x>
      <y>320</y>
      <width>400</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="selectRootPath">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>69</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="selectWorkshopPath">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>200</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="selectSteamPath">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>320</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
